src/Classes/WorkerTaxes/Payroll.php

```diff
@@ -144,28 +148,54 @@ public function getYtdEarnings(GovernmentalUnitArea $governmental_unit_area = nu
             return 0;
         }
 
-        return $this->wage_manager->calculateEarnings($area_income->getHistoricalWages(),
-            $this->start_date->copy()->startOfYear());
+        return $this->wage_manager->calculateEarnings(
+            $area_income->getHistoricalWages(),
+            $this->start_date->copy()->startOfYear()
+        );
+    }
+
+    public function getTipAmount(GovernmentalUnitArea $governmental_unit_area = null)
+    {
+        if ($governmental_unit_area === null) {
+            return 0;
```

Comment:
this should probably return `$this->tip_amount`, which would get set in `determineEarnings`